libs/grok/src/main/resources/patterns/grok-patterns

```diff
@@ -62,7 +62,7 @@ MINUTE (?:[0-5][0-9])
 # '60' is a leap second in most time standards and thus is valid.
 SECOND (?:(?:[0-5]?[0-9]|60)(?:[:.,][0-9]+)?)
 TIME (?!<[0-9])%{HOUR}:%{MINUTE}(?::%{SECOND})(?![0-9])
-# datestamp is YYYY/MM/DD-HH:MM:SS.UUUU (or something like it)
+# datestamp is yyyy/MM/DD-HH:MM:SS.UUUU (or something like it)
```

Comment:
I think the `YYYY` here is simply for showing the format, as the other letters are all upper case, but do not have a meaning (like the `u`)